x-pack/plugins/apm/common/instances.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import * as t from 'io-ts';
+
+export const instancesSortFieldRt = t.keyof({
+  serviceNodeName: null,
+  latency: null,
+  throughput: null,
+  errorRate: null,
+});
```

Comment:
I think using `t.literal` is more semantically correct than `t.keyof`

```suggestion
export const instancesSortFieldRt = t.union([
  t.literal('serviceNodeName'),
  t.literal('latency'),
  t.literal('throughput'),
  t.literal('errorRate'),
]);
```

Reply:
The [docs](https://github.com/gcanti/io-ts/blob/master/index.md#union-of-string-literals) recommend to use `t.keyof` instead of `t.union`